### ISSUE TITLE
Conditional specs aren't being resolved and result in an error

### ISSUE THREAD

**Issue by rrijnberk** [mdasberg/grunt-karma-sonar#34]
When using conditional test names the plugin is unable to resolve them to a spec file. Instead of breaking the process don't set the classname.

**Comment by rrijnberk** [comment]
i.e.
```js
for(x =0; x < 10; x++) {
   it('should do something with ' + x, function() {
      ....
   });
}
```

**Comment by mdasberg** [comment]
@see https://github.com/mdasberg/grunt-karma-sonar/pull/35

**Comment by mdasberg** [comment]
fixed in release 0.2.17
